packages/core/src/provider/base.ts

```diff
@@ -231,7 +231,7 @@ export class SmartAccountProvider<
     );
   };
 
-  sendTransactions = async (requests: RpcTransactionRequest[]) => {
+  buildUserOperationFromTxs = (requests: RpcTransactionRequest[]) => {
```

Comment:
what's changed here? is this new?

Reply:
just refactored out the building of the uoStruct to another method, still being called in `sendTransactions`